### PR TITLE
LITE-29281: Add Feed Details view

### PR DESCRIPTION
**Pull request by arnaugiralt**
Add Feed Details view:
- Create `FeedDetails` component
- Create `LoadingIndicator` component
- Create `FeedActions` component
- Implement `FeedActions` in `FeedsView` component
- Updated `feeds.details` route component
- Create `DetailItem` component
- Create `DetailItemGroup` component
- Add more color constants
- Add styles for a horizontal divider
- Create api helpers to handle API requests
- Create api composable to handle API requests inside Vue components

---

**extra** Updated ESLint config, imports are sorted now.
- Added `eslint-plugin-import`
- Updated dependencies
- Fixed ESLint issues with the new configuration

---

### Screenshots

Feeds view with the feed actions:
Feeds view with the feed actions

Feed Details:
Feed details

Actions if feed is enabled:
Actions if feed is enabled

Actions if feed is disabled:
Actions if feed is disabled

The SPA snackbar displays errors if a request fails:
The SPA snackbar displays errors if a request fails

### CODE REVIEW

**Review by maxipavlovic — DISMISSED** [review]

File: ui/components/FeedActions.vue [hunk 1]
```diff
@@ -0,0 +1,179 @@
...
+
+const buttons = reactive([
+  {
+    action: 'disable',
+    color: COLORS_DICT.TEXT,
+    text: 'Disable',
+    icon: 'googleToggleOffBaseline',
+    loading: disableAction.loading,
+    hide: isFeedDisabled,
+    handler: async () => {
```
Comment (nerimartinez) [inline ui/components/FeedActions.vue:82]: wdyt about moving this into a different method?

File: ui/components/FeedActions.vue [hunk 2]
```diff
@@ -0,0 +1,179 @@
...
+    color: COLORS_DICT.TEXT,
+    text: 'Force upload',
+    icon: 'googleUploadBaseline',
+    loading: forceUploadAction.loading,
+    disabled: isFeedDisabled.value,
+    handler: async () => {
+      try {
+        await forceUploadAction.request(`/api/feeds/${feedId.value}/uploads/force`, 'POST');
+        emit('uploaded');
+      } catch (e) {
```
Comment (nerimartinez) [inline ui/components/FeedActions.vue:118]: maybe get the status code? and add an if

File: ui/components/FeedActions.vue (hunk 1 above)
Comment (arnaugiralt) [reply ui/components/FeedActions.vue]: Good point, I extracted all handlers outside the `buttons` variable

File: ui/components/FeedActions.vue (hunk 2 above)
Comment (arnaugiralt) [reply ui/components/FeedActions.vue]: Yep, looks way better now in my opinion. Now the `useRequest`'s `request` method always returns the response's status code.

**Review by nerimartinez — APPROVED** [review]

**Pull request merged**